include/swift/Basic/Compiler.h

```diff
@@ -178,4 +178,11 @@
 #define SWIFT_VFORMAT(fmt)
 #endif
 
+// Tells Swift's ClangImporter to import a C++ type as a foreign reference type.
+#if __has_attribute(swift_attr)
+#define SWIFT_IMPORT_REFERENCE __attribute__((swift_attr("import_as_ref")))
```

Comment:
@zoecarver when you'll be changing the spelling of `import_as_ref`, could you please add the new spelling here as a second attribute without removing `import_as_ref`? That would help to avoid breaking the hosttools build.